packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html';
+
+import '../ui/elements.dart';
+
+void listenForDragScrolling(CoreElement element) {
```

Comment:
nit: change method name.
this is actually making the CoreElement scroll via dragging instead of just listening.
So maybe:
void enableDragScrolling
or
void initalizeDragScrolling
?

Reply:
good point. changed to enableDragScrolling.